src/screens/HomeScreen.js

```diff
@@ -77,29 +79,39 @@ export const HomeScreen = ({ navigation }) => (
                   message: shareMessage(newsItem, 'newsItem')
                 }
               },
+              bottomDivider: index === data.newsItems.length - 1 ? false : true,
```

Comment:
We can upgrade the condition to only `index === data.newsItems.length - 1`, because if this is false, it returns false and if it is true it returns true. We do not have to write it explicitly.